src/ol/reproj/DataTile.js

```diff
@@ -116,7 +116,7 @@ class ReprojDataTile extends DataTile {
 
     /**
      * @private
-     * @type {!Array<import("../Tile.js").default>}
+     * @type {!Array<DataTile>}
```

Comment:
The `reproject_` method below assumes these are data tiles, but TypeScript didn't notice with the previous syntax.